### ISSUE TITLE
Twilio gateway compatibility issue with Twilio 6.0 client

### ISSUE THREAD

**Issue by wesvetter** [Bouke/django-two-factor-auth#211]
I'm getting an error using 1.6.1 with the Twilio 6.0 library:

```
  File ".../two_factor/models.py", line 116, in generate_challenge
    send_sms(device=self, token=token)
  File ".../two_factor/gateways/__init__.py", line 16, in send_sms
    gateway.send_sms(device=device, token=token)
  File ".../two_factor/gateways/twilio/gateway.py", line 66, in send_sms
    self.client.sms.messages.create(
AttributeError: 'Client' object has no attribute 'sms'
```

It seems that changing `client.sms.messages.create(...)` to `client.messages.create(...)` fixes the issue and behaves as expected (ie. I receive my SMS token). I've made a commit to that effect in [my fork][fork].

I can submit a PR for this but I'm not really familiar with this library or the Twilio one, and don't want to break compatibility for anyone else. Do we need to check which library is installed first and then call the appropriate method?

[fork]:https://github.com/wesvetter/django-two-factor-auth/commit/c3151346425bb89d8f59f7d6b7258207bb017749

**Comment by Bouke** [comment]
Hi Wes, thank you for the report. I believe the sms endpoint was deprecated some time ago, although I don't think the client warned about this. I think we should just replace the call altogether, and set a minimum version for the client. It would be great if you could provide a PR.

**Comment by PierreF** [comment]
Could you make a new release with this fix ?

Django-two-factor-auth 1.6.1 can't sent SMS at all with twilio, because the sms endpoint was removed with twilio 6.0 and DTFA 1.6.1 needs twilio >= 6.0.

**Comment by Bouke** [comment]
I've pushed 1.6.2 with the fix of #212 to pypi.
